MC/run/PWGGAJE/run_dirgamma_embedding.sh

```diff
@@ -55,13 +55,15 @@ echo 'Detector acceptance option ' $PARTICLE_ACCEPTANCE
 export CONFIG_OUTPARTON_PDG=${CONFIG_OUTPARTON_PDG:-0}
 
 # create workflow
-${O2DPG_ROOT}/MC/bin/o2dpg_sim_workflow.py -eCM ${CONFIG_ENERGY} -col pp -gen pythia8 -proc "dirgamma" \
-                                            -ptHatMin ${PTHATMIN} -ptHatMax ${PTHATMAX}                \
-                                            -tf ${NTIMEFRAMES} -ns ${NSIGEVENTS} -e ${SIMENGINE}       \
-                                            -nb ${NBKGEVENTS} --embedding                              \
-                                            -j ${NWORKERS} -mod "--skipModules ZDC"                    \
-                                            -weightPow ${WEIGHTPOW}                                    \
-                                            -trigger "external" -ini "\$O2DPG_ROOT/MC/config/PWGGAJE/ini/trigger_prompt_gamma.ini"
+${O2DPG_ROOT}/MC/bin/o2dpg_sim_workflow.py -eCM ${CONFIG_ENERGY}                                     \
+                                           -nb ${NBKGEVENTS} --embedding                             \
+                                           -colBkg PbPb -genBkg pythia8 -procBkg "none"              \
```

Comment:
what does "none" mean as process? Is it different from `inel`?

Reply:
It means no process specified, apparently Pb-Pb Pythia does not need the process

Reply:
@gconesab 
PbPb needs more than just not setting the process.
I suggest that you use process `heavyion` as a label that adds the required settings for pythia8 heavy-ion machinery

Reply:
in mkpy8cfg.py I activate those when Pb-Pb is set, but I agree it is better to specify a HI case, I will do that